notebooks/115-async-api/README.md

```diff
@@ -0,0 +1,16 @@
+# Runtime Inference Optimization with Async API
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F115-async-api%2F115-async-api.ipynb)
+
+
+This notebook demostrate how to use [asynchronous execution](https://docs.openvino.ai/latest/openvino_docs_deployment_optimization_guide_common.html) to improve data pipelining. 
+
+OpenVINO™ Runtime supports inference in either synchronous or asynchronous mode. Using the Async API can improve application’s overall frame-rate, because thie function can keep the hardware busy by sending inference requests continuously. In other words, if you can afford sending a new request without waiting for the previous one to finish. 
```

Comment:
```suggestion
OpenVINO Runtime supports inference in either synchronous or asynchronous mode. The key advantage of the Async API is that when a device is busy with inference, the application can perform other tasks in parallel (e.g. populating inputs or scheduling other requests) rather than wait for the current inference to complete first. 
```